app/models/ems_refresh/save_inventory_infra.rb

```diff
@@ -214,6 +214,13 @@ def save_hosts_inventory(ems, hashes, target = nil)
     end
   end
 
+  def save_hosts_storages_inventory(host, hashes, target = nil)
+    hashes.each do |mount|
+      storage_id = mount[:storage][:id]
+      HostsStorage.where(:host_id => host.id, :storage_id => storage_id).update_all(:read_only => mount[:read_only])
```

Comment:
Ah interesting...it assumes a `nil`...didn't realize that.  I'm not sure we should add an id column, but then again, if it's getting real data in it, you may want to access it.  Hmmm